tests/test_api_client.py

```diff
@@ -0,0 +1,49 @@
+import os
+import unittest
+from unittest import mock
+
+from artapi.client import CoreApiClient
+
+
+class TestResponse:
+    def __init__(self):
+        self.status_code = 401
+
+
+class TestApiClient(unittest.TestCase):
```

Comment:
There is no tests for: is_token_expired, expires_at, _get_token.
Also you can came up with something to test poe, poa, statuses.